src/frontend/apps/standalone_site/src/components/Layout/MainLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import { theme } from 'lib-common';
+import React from 'react';
+import styled from 'styled-components';
+
+const colorMenu = normalizeColor('blue-active', theme);
+
+const MainLayoutBox = styled(Box)`
+  color: ${colorMenu};
+  min-height: 100vh;
+`;
+
+const MainBox = styled(Box)`
+  flex: 1;
+`;
+
+interface MainLayoutProps {
+  children: React.ReactNode;
+  menu: React.ReactNode;
+}
+
+function MainLayout({ children, menu }: MainLayoutProps) {
```

Comment:
```suggestion
interface MainLayoutProps {
  menu: React.ReactNode;
}

function MainLayout({ children, menu }: React.PropsWithChildren<MainLayoutProps>) {
```

Reply:
It nice, I didnt know it, however in this case children should be mandatory, PropsWithChildren make it optional.

Reply:
Yeah but is there any reason for the children to be mandatory ?

Reply:
😄 Doesn't make sense this component without children.